geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/AlterMappingCommand.java

```diff
@@ -100,27 +102,24 @@ public Result alterMapping(
     // action
     List<CliFunctionResult> results =
         executeAndGetFunctionResult(new AlterMappingFunction(), newMapping, targetMembers);
-
-    boolean persisted = false;
-    // update the cc with the merged connection returned from the server
-    if (ccService != null && results.stream().filter(CliFunctionResult::isSuccessful).count() > 0) {
-      ConnectorService service =
-          ccService.getCustomCacheElement("cluster", "connector-service", ConnectorService.class);
-      if (service == null) {
-        service = new ConnectorService();
-      }
-      CliFunctionResult successResult =
-          results.stream().filter(CliFunctionResult::isSuccessful).findAny().get();
-      ConnectorService.RegionMapping mergedMapping =
-          (ConnectorService.RegionMapping) successResult.getResultObject();
-      CacheElement.removeElement(service.getRegionMapping(), connectionName);
-      service.getRegionMapping().add(mergedMapping);
-      ccService.saveCustomCacheElement("cluster", service);
-      persisted = true;
-    }
-
     CommandResult commandResult = ResultBuilder.buildResult(results);
-    commandResult.setCommandPersisted(persisted);
+
+    // find the merged regionMapping from the function result
```

Comment:
I'd keep the casing consistent with the object to which it refers, i.e., `RegionMapping` instead of `regionmapping`.

---

This comment was incorrectly labeled as "outdated," possibly also a result of a force-push update.